src/python/pants/engine/subsystem/native_engine_version

```diff
@@ -1 +1 @@
-e5b11346a458240934cc6136e40faa62b75f79ff
+da39a3ee5e6b4b0d3255bfef95601890afd80709
```

Comment:
might be wrong, but I'd only expect this to get modified with rust code changes?